admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -121,73 +99,70 @@ Enabling Master Key Based Encryption
 
 To enable master key based encryption:
 
-1. Enable the default encryption module app, using the following command::
+1. Enable the default encryption module app, using the following command
+
+::
 
   php occ app:enable encryption
 
-2. Then enable encryption, using the following command :: 
+2. Then enable encryption, using the following command 
+
+:: 
 
   php occ encryption:enable
 
-3. Then enable the master key, using the following command::
+3. Then enable the master key, using the following command
+
+::
 
   php occ encryption:enable-master-key
 
 .. note::
 
-   Master key mode has to be setup in a newly created instance.
+   The master key mode has to be set up in a newly created instance.
    
 Sharing Encrypted Files
 -----------------------
 
-After encryption is enabled your users must also log out and log back in to 
-generate their personal encryption keys. They will see a yellow warning banner 
-that says "Encryption App is enabled but your keys are not initialized, please 
-log-out and log-in again." 
+After encryption is enabled, your users must also log out and log back in to generate their personal encryption keys. 
+They will see a yellow warning banner that says "*Encryption App is enabled, but your keys are not initialized. Please log-out and log-in again.*" 
 
-Share owners may need to re-share files after encryption is enabled; users 
-trying to access the share will see a message advising them to ask the share 
-owner to re-share the file with them. For individual shares, un-share and 
-re-share the file. For group shares, share with any individuals who can't access 
-the share. This updates the encryption, and then the share owner can remove the 
-individual shares.
+Also, share owners may need to re-share files after encryption is enabled. 
```

Comment:
Not a question for this (mostly formatting... review). But this seems an odd statement. Surely there is an algorithm that determines under what conditions the enabling of encryption will cause existing shares to become unusable. Or maybe they always become unusable, or?